contributors/guide/issue-triage.md

```diff
@@ -94,178 +100,208 @@ Some of its features:
 
 If you can't reproduce it:
-* Contact the issue reporter with your findings 
+* Contact the issue reporter with your findings .
 * Close the issue if both the parties agree that it could not be reproduced.
 
 If you need more information to further work on the issue:
 * Let the reporter know it by adding an issue comment. Include `/triage needs-information` in the comment to apply the `triage/needs-information` label.
 
-In all cases, if you do not get a response in 20 days then close the issue with an appropriate comment. If you have permission to close someone else's issue, first `/assign` the issue to yourself, then `/close` it. If you do not, please leave a comment describing your findings.
+In all cases, if you do not get a response within 20 days, close the issue with an appropriate comment. If you have permission to close someone else's issue, first `/assign` the issue to yourself, then `/close` it. If you do not, please leave a comment describing your findings.
 
 ### Help Wanted/Good First Issues
+
 To identify issues that are specifically groomed for new contributors, we use the [help wanted](https://github.com/kubernetes/kubernetes/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22)
 and [good first issue](https://github.com/kubernetes/kubernetes/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) labels. To use these labels:
 * Review our specific [guidelines](/contributors/guide/help-wanted.md) for how to use them.
-* If the issue satisfies these guidelines, you can add the `help wanted` label with the `/help` command
+* If the issue satisfies these guidelines, you can add the `help wanted` label with the `/help` command.
 and the `good first issue` label with the `/good-first-issue` command. Please note that adding the `good first issue` label will also automatically add the `help wanted` label.
 * If an issue has these labels but does not satisfy the guidelines, please ask for more details to be added to the issue or remove the labels using the `/remove-help` or `/remove-good-first-issue` commands.
 
 ### Kind Labels
+
 Usually the `kind` label is applied by the person submitting the issue. Issues that feature the wrong `kind` (for example, support requests labelled as bugs) can be corrected by someone triaging; double-checking is a good approach. Our [issue templates](https://github.com/kubernetes/kubernetes/issues/new/choose) aim to steer people to the right kind.
 
 ## Step Three: Define Priority
+
 We use GitHub labels for prioritization. If an issue lacks a `priority` label, this means it has not been reviewed and prioritized yet. 
 
 We aim for consistency across the entire project. However, if you notice an issue that you believe to be incorrectly prioritized, please leave a comment offering your counter-proposal and we will evaluate it.
 
+{{< table caption="Priority labels for issues" >}}
```

Comment:
```suggestion
<!--- NOTE: Table syntax is for the Hugo rendered page -->
{{< table caption="Priority labels for issues" >}}
```

Similar thing here too, just trying to prevent future confusion^^:;;